src/apis/blockchain_api.ts

```diff
@@ -71,6 +82,10 @@ export class BlockchainAPI {
         const intervalManager = this.intervalManager;
         const web3Utils = this.web3Utils;
 
+        if (!_.isNumber(timeoutMs)) {
```

Comment:
why not specify this default value in the signature and remove the optional

Reply:
Oh I forgot you could do that lol

Reply:
that's what friends are for!